### ISSUE TITLE
Homepage image change for Phase II Redesign

### ISSUE THREAD

**Issue by apinkney0696** [sciencehistory/scihist_digicoll#2140]
Hello! I put together some options for the header images. There are a few variations ranging from no edits to some playful edits. You can find options here: P:\Othmer Library\Digital Collections - Internal Access\Website Updates\Homepage\Header Images\JPG OPTIONS

I also have the Photoshop files for all of these, so I can change them as we'd like. I'm thinking we use the following:

BettyeB.jpg
Header_Option_2a copy.jpg
Header_Option_5 copy.jpg
Header_Option_6 copy.jpg
Header_Option_7 copy.jpg
and Header_Option_8 copy.jpg

although I would love to preview what each of these looks like on the new staging site before making a final decision. Let me know if these are the right dimensions too!

**Comment by jrochkind** [comment]
@apinkney0696 

You're caling these "header images", but theyr'e definitely for hte home page on the top left, correct?  That is not an area I call the "header", but we may use different language which is fine!

Currently in that location, the image has a little caption "Wilson College chemistry club", which then links to the relevant Work. The main site redesign does _not_ generally do this when using images from the collection. 

If you'd like to continue doing that, please supply the Work that each image came from so i can make it have the caption/link.  If not, then not!  I have no real opinion on whether we should or not.

**Comment by jrochkind** [comment]
I am not sure people would know to click on the image without a label; if we want people to be able to get from the image to the source record, to me it's more accessible to provide the label as a hyperlink/caption, the same as we are now.  the image as an unlabelled hyperlink does not seem right to me. 

But it's technically possible either way, sure! 

Are those links in the same order as the images you listed?  If you had a moment, it would make things easier for me and skip the step of corresponding if you just provided a list of the images and the URLs together!

**Comment by apinkney0696** [comment]
Good call - we should keep the caption for accessibility then, but I do think that the image should also be linked!

The image files I have listed are just the ones I'd like to test first, but not necessarily the final options. As I mentioned above, I'd like to preview all the images included in the folder and potentially do some additional editing before deciding on the final options. Would you like the links corresponding to all 11 image options now or would it be easier for you to do the linking after I make the final selections? 

We should probably set up a meeting to go through and look at the image options in real time!

**Comment by jrochkind** [comment]
You had mentioned you wanted to preview them... I had imagined that to allow you to do that, we'd put in a (possibly temporary) feature where if you put like `&hero_image=1` in the URL, it would choose a specific image. From `1` to how many images there are. 

I can't really think of a better way to allow you to preview then that, even if we did meet realtime. 

So maybe give us a list of all the images you'd like to be available to preview, with work URL, and the title of the work too if you have it there in front of you and want to save us the time of looking it up ourselves? And then we'll put them all in a staging branch so they can be previewed in the context of the new feature. Make sense?  

I don't plan to get to this until after the rest of Phase 2, which will be a couple few weeks at least, so no rush.
